upgrading.md

```diff
@@ -39,6 +37,32 @@ Follow these steps to update the new package repository:
       --namespace tap-install
     ```
 
+## <a id="upgrading-to-version-11"></a> Upgrading to Version v1.1.0
```

Comment:
The anchor for this header reads like "Upgrading to Version 11". Could we change it to "upgrading-to-version-1-1"?